src/components/StatusDashboard/repos_and_bots.jsx

```diff
@@ -57,12 +57,12 @@ function Image({ alt, link, children }) {
 }
 
 function CDNStatus() {
-  const [{ minutes, status }, setState] = useState({ minutes: 0, status: "…" });
+  const [{ minutes, status }, setState] = useState({ minutes: "…", status: "…" });
   useEffect(() => {
     void (async () => {
       try {
-        const response = await (await fetch(urls.repos.cdn.api)).text();
-        const updated = new Date(response.trim()).getTime();
+        const response = await (await fetch(`${urls.repos.cdn.api}?bustcache=${Date.now()}`)).json();
```

Comment:
Weird that we need this `bustcache` trick again (also needed in the [packages page for the RSS stuff](https://github.com/conda-forge/conda-forge.github.io/blob/8399abe74226fc690d9b8e37e4083c9039452158/src/components/Packages/index.jsx#L82-L87)). Is this a `conda.anaconda.org` misconfiguration? We don't need it for e.g. `raw.githubusercontent.com`.

cc @jezdez @morremeyer

Reply:
@jaimergp I'll look into how we can improve that this week. We explicitly purge all updated files from the Cloudflare cache at the end of each cache run.

However, we do also set a `Cache-Control: public, max-age=1200` header in responses so that clients don't always download the files anew. 

I expect that this leads to the effect that the status does not update: If the browser requests the file when the last update was 19 minutes ago, it will serve it from cache for the next 20 minutes, so it will show the last update being 39 minutes ago even though there should have been another update in the meantime.

Reply:
Maybe the `max-age` for these status files (`last_updated.json`, `rss.xml`, etc) should be smaller, like a minute or something? I'm busting the browser cache with these `?bustcache=<timestamp>` hacks but ideally we wouldn't need them.

Reply:
Yup, my thoughts exactly.

Reply:
I'd stick with the work-around for now.